homeassistant/components/group/__init__.py

```diff
@@ -255,6 +256,12 @@ async def groups_service_handler(service):
                 or None
             )
 
+            if service.data.get(ATTR_REMOVE_ENTITIES) is not None:
+                _LOGGER.warning(
+                    "%s: remove_entities ignored when creating a new group",
+                    service.service,
+                )
+
```

Comment:
Let's not print this, as it prevents people from using same service call every time. 
```suggestion
```

Reply:
If creating a new group and specifying remove_entities is acceptable usage, it seems incorrect to silently ignore the specified remove_entities. If not printing this warning, should we add support for removing entities from the new group?